app/src/view/screens/SearchProducts.js

```
@@ -21,501 +549,411 @@ import AsyncStorage from "@react-native-async-storage/async-storage";
+    return;
+  }
+  const searchTerm = text.toLowerCase();
 
-    if (matchingItem) {
-      setSelectedItemId(matchingItem.title);
-    } else {
-      setSelectedItemId(null);
-    }
-  };
+  const filteredApps = data.filter(
+    (app) => app.title && app.title.toLowerCase().includes(searchTerm)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)